lib/Onyx.js

```diff
@@ -639,8 +680,9 @@ function keyChanged(key, data, canUpdateSubscriber, notifyRegularSubscibers = tr
  * @param {String} [mapping.selector]
  * @param {*|null} val
  * @param {String} matchedKey
+ * @param {Boolean} batched
  */
-function sendDataToConnection(mapping, val, matchedKey) {
+function sendDataToConnection(mapping, val, matchedKey, batched) {
```

Comment:
For the boolean params it might be better to write them as `isBatched`
```suggestion
function sendDataToConnection(mapping, val, matchedKey, isBatched) {
```